projects/plugins/jetpack/tests/e2e/specs/recommendations.test.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Internal dependencies
+ */
+import { step } from '../lib/env/test-setup';
+import RecommendationsPage from '../lib/pages/wp-admin/recommendations';
+
+describe( 'Jetpack Recommendations', () => {
+	it( 'Steps', async () => {
```

Comment:
This is the name of the test. Maybe something like "Jetpack Recommendations" would be more suggestive. It can be the same as the suite (describe block).